content/blog/cursed-container-iceberg/index.md

```diff
@@ -0,0 +1,157 @@
+---
+title: "The Cloud Container Iceberg"
+date: 2024-12-03T10:29:54-05:00
+draft: false
+meta_desc: Discover unique and unconventional ways to run OCI containers. Explore beyond the traditional platforms.
+meta_image: meta.png
+authors:
+    - adam-gordon-bell
+tags:
+    - k8s
+social:
+    twitter: There are 105+ ways to run containers in production! From familiar cloud providers to quantum computers, smart toasters, and even phone farms. 
+    linkedin: There are 105+ ways to run containers in production! From familiar cloud providers to quantum computers, smart toasters, and even phone farms. 
+
+---
+If a computer can be connected to the internet, someone has tried to run a container on it. From quantum computers to smart toasters, from phones in AWS racks to CI pipelines, there's many ways to deploy containers. While most people only know about the mainstream cloud providers at the surface, there's actually a vast world of increasingly unusual and specialized options beneath.
```

Comment:
The 2nd paragraph is a bit of run on. It has 2 sentence fragments stacked together